data/fieldlist_NCAR.jsonc

```diff
@@ -157,6 +156,22 @@
       "units": "W m-2",
       "ndim": 3
     },
+    // Variables for flow_dep_diag module:
```

Comment:
@jkrasting Is this then not something I need to worry about?

Reply:
@drewmresnick This is a todo for the framework team, so one of us will take care of it.

Reply:
Thanks for bringing this to my attention. Unfortunately constructions of the from `variable_at_XX_mbar` aren't CF standard names [*]. 

The way variables on pressure levels are intended to be specified in the settings file is described in the [documentation](https://mdtf-diagnostics.readthedocs.io/en/latest/sphinx/ref_settings.html#varlist-entry-properties): the level is specified separately with a `scalar_coordinate` attribute, which refers to a vertical coordinate axis defined in the [dimensions](https://mdtf-diagnostics.readthedocs.io/en/latest/sphinx/ref_settings.html#dimensions-section) section of the file. By spelling out what's meant in more detail like this, it e.g. enables the framework to extract levels from 3D data on the fly in a fully units-aware way. 

For more examples of how this is set up, look at the [settings file](https://github.com/NOAA-GFDL/MDTF-diagnostics/blob/a7b13c2311ca7af10dc89f00a35d6e50c3903a91/diagnostics/Wheeler_Kiladis/settings.jsonc#L54-L64) for the Wheeler-Kiladis diagnostic.

[*]: I haven't been able to find a concise reference for this: the upshot is that standard names are meant to identify [physical quantities](http://cfconventions.org/cf-conventions/cf-conventions.html#standard-name), independently of how that quantity is sampled in space or time (but there are exceptions to this, such as `air_pressure_at_sea_level`, which deal with physical rather than mathematical boundaries). Variables on pressure levels in the CMIP6 data request don't use this construction, and pressure levels aren't one of the cases covered in [construction of derived standard names](https://cfconventions.org/Data/cf-standard-names/docs/guidelines.html).